From 4f682ef5a6fb47f43047b5be5dc330aa36655508 Mon Sep 17 00:00:00 2001
From: Meir Levi <mlevi4@marvell.com>
Date: Thu, 24 Dec 2020 10:15:22 -0500
Subject: [PATCH 1064/1921] Documentation: devicetree: bindings: net:
 marvell-musdk-cma documentation

Marvell MUSDK continuous memory allocation driver (BM) documentation

Signed-off-by: Meir Levi <mlevi4@marvell.com>
Change-Id: I44d228217e3816ef454fb8260e7491892ea73241
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/42774
Tested-by: Stefan Chulski <stefanc@marvell.com>
Reviewed-by: Stefan Chulski <stefanc@marvell.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 .../bindings/net/marvell-musdk-cma.txt           | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)
 create mode 100644 Documentation/devicetree/bindings/net/marvell-musdk-cma.txt

diff --git a/Documentation/devicetree/bindings/net/marvell-musdk-cma.txt b/Documentation/devicetree/bindings/net/marvell-musdk-cma.txt
new file mode 100644
index 000000000000..4aa66bbdce5c
--- /dev/null
+++ b/Documentation/devicetree/bindings/net/marvell-musdk-cma.txt
@@ -0,0 +1,16 @@
+* Marvell MUSDK continuous memory allocation driver (BM)
+
+Required properties:
+
+- compatible: should be "marvell,musdk-cma".
+
+Optional properties:
+
+- dma-coherent
+
+Example:
+
+musdk_cma {
+	compatible = "marvell,musdk-cma";
+	dma-coherent;
+};
-- 
2.31.1

